LXReorderableCollectionViewFlowLayout/LXReorderableCollectionViewFlowLayout.m

```diff
@@ -299,7 +301,7 @@ - (void)handleLongPressGesture:(UILongPressGestureRecognizer *)gestureRecognizer
             collectionViewCell.highlighted = NO;
             UIView *imageView = [collectionViewCell LX_snapshotView];
             imageView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
-            imageView.alpha = 0.0f;
+            imageView.alpha = 1.0f;
```

Comment:
This also seems like a substantial behavior change for other use cases.   Maybe make it conditional on the presence of `reorderStartAnimationBlock`